### DISCUSSION TITLE
Terminal in Nautilus

### DISCUSSION THREAD

**Discussion by git4maz** [mylinuxforwork/dotfiles#310] [General]
Hi Stephan,
thanks for your project, that's awesome.
I want just to tell you that open terminal in nautilus actually doesn't work.Cordially

**Comment by mylinuxforwork** [comment]
Thanks for your feedback. Have you installed 2.9.6.3?

**Comment by git4maz** [comment]
Yes,I've installed 2.9.6.3

Il 10 ottobre 2024 13:06:00 CEST, Stephan Raabe ***@***.***> ha scritto:

**Reply by mylinuxforwork** [reply]
You need to reboot to activate it. Have you done a reboot after the update? And dod you run ml4w-hyprland-setup?

**Comment by mylinuxforwork** [comment]
Do you see this menu entry? 
![image](https://github.com/user-attachments/assets/82fdfa2a-ed67-436c-b554-cfc7b2f8c445)

**Comment by git4maz** [comment]
Yes,I see that but It doesn't display anything

Il 10 ottobre 2024 15:03:29 CEST, Stephan Raabe ***@***.***> ha scritto:

**Comment by git4maz** [comment]
And I ran ml4w-hyprland-setup to try

Il 10 ottobre 2024 14:58:44 CEST, Stephan Raabe ***@***.***> ha scritto:

**Comment by git4maz** [comment]
I've done the update one more time and now It works fine!!Thanks a lot for
your help and work.
Have a nice day!

Il giorno gio 10 ott 2024 alle ore 15:09 Andrea Mazzola ***@***.***>
ha scritto:
